tests/cases/compiler/pushTypeGetTypeOfAlias.ts

```diff
@@ -0,0 +1,9 @@
+// @checkJs: true

```

Comment:
```suggestion
// @checkJs: true
// @allowJs: true
// @noEmit: true
```

To avoid the errors above.